WordPress/src/main/java/org/wordpress/android/modules/ApplicationModule.java

```diff
@@ -1,15 +1,40 @@
 package org.wordpress.android.modules;
 
 import android.app.Application;
```

Comment:
Let's clean up unused imports such as this one in this class.

I'd recommend enabling the "Optimize imports on the fly" setting in AS. Although not 100% reliable, it works in most situations.

Reply:
Fixed - thanks for the tip;).